src/lecturer/components/sendQuizView/SendQuizView.tsx

```diff
@@ -154,30 +167,26 @@ export function SendQuizView(props: SendQuizViewProps) {
             case 1:
                 return (
                     <FormControl className={classes.container}>
-                        <InputLabel id="time-select">
-                            Czas na wypełnienie
-                        </InputLabel>
-                        <Select
-                            labelId="time-select"
-                            className={classes.textField}
-                            id="time-select"
-                            value={time > 0 ? time : ""}
-                            onChange={handlePicker}
-                            disabled={checked}
-                            autoFocus={!checked}
-                        >
-                            <MenuItem value={0.5}>30 sek</MenuItem>
-                            <MenuItem value={1}>1 min</MenuItem>
-                            <MenuItem value={2}>2 min</MenuItem>
-                            <MenuItem value={3}>3 min</MenuItem>
-                            <MenuItem value={5}>5 min</MenuItem>
-                            <MenuItem value={10}>10 min</MenuItem>
-                            <MenuItem value={20}>20 min</MenuItem>
-                            <MenuItem value={40}>40 min</MenuItem>
-                            <MenuItem value={60}>1 h</MenuItem>
-                            <MenuItem value={90}>1.5 h</MenuItem>
-                            <MenuItem value={180}>3 h</MenuItem>
-                        </Select>
+                        <Grid container spacing={3} style={{ margin: "0 0 5px 0" }}>
+                            <Grid item xs={3} style={{ paddingLeft: "0" }}>
+                                <TextField
+                                    label="Minuty"
+                                    value={minutes >= 0 ? minutes : ""}
+                                    onChange={handleMinutesChange}
+                                    disabled={checked}
+                                    autoFocus={!checked}
+                                />
+                            </Grid>
+                            <Grid item xs={3}>
+                                <TextField
+                                    label="Sekundy"
+                                    value={seconds >= 0 ? seconds : ""}
+                                    onChange={handleSecondsChange}
+                                    disabled={checked}
+                                    autoFocus={!checked}
+                                />
```

Comment:
```suggestion
                                <TextField
                                    label="Minuty"
                                    type="number"
                                    value={minutes >= 0 ? minutes : ""}
                                    onChange={handleMinutesChange}
                                    disabled={checked}
                                    autoFocus={!checked}
                                />
                            </Grid>
                            <Grid item xs={3}>
                                <TextField
                                    label="Sekundy"
                                    type="number"
                                    value={seconds >= 0 ? seconds : ""}
                                    onChange={handleSecondsChange}
                                    disabled={checked}
                                    autoFocus={!checked}
                                />
```
Add type to inputs so that there will be arrows for increasing the numbers.
![image](https://user-images.githubusercontent.com/26445545/119273680-43ea7380-bc0c-11eb-8ea5-1dcc8fcf764d.png)